core/src/main/scala/neotypes/CypherStringInterpolator.scala

```diff
@@ -17,25 +18,33 @@ object CypherStringInterpolator {
     val params = parameters.iterator
 
     @annotation.tailrec
-    def loop(paramNext: Boolean, output: List[DeferredQueryBuilder.Part]): List[DeferredQueryBuilder.Part] =
+    def loop(paramNext: Boolean, nextSubQuery: Int, output: List[DeferredQueryBuilder.Part]): List[DeferredQueryBuilder.Part] =
```

Comment:
Not related to your change, but since this method does a lot of concatenations I have thought of changing it to internally use either a `Vector` or a `ListBuilder`; of course, the return will still be a List`
WDYT?

Reply:
Not sure. I must say that I sadly have very little concept of the performance implications. Coming from a world where mutability is not even an option I usually don't even consider `ListBuilder` :D Unless I'm wrong however this would only affect compile-time performance, right?

Reply:
_(for some reason I can't tag you with @ oh well, hope you see it)_

> I usually don't even consider ListBuilder :D

Yeah, I also try to avoid mutability in my code, but a local-scoped `ListBuilder` doesn't hurt :)

> Unless I'm wrong however this would only affect compile-time performance, right?

But, I believe you are right. This is why I never bothered to make the change myself, I just decided to mention it today to see what you think.